build_debian.sh

```diff
@@ -260,6 +260,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT umount /proc
 ## Prepare empty directory to trigger mount move in initramfs-tools/mount_loop_root, implemented by patching
 sudo mkdir $FILESYSTEM_ROOT/host
 
+## Create loop device for /var/log to limit its size to 1000000*512=500MB
+sudo mkdir -p $FILESYSTEM_ROOT/var/disk-img
+sudo LANG=C chroot $FILESYSTEM_ROOT dd if=/dev/zero of=/var/disk-img/var-log.ext4 count=1000000
```

Comment:
the question here is that say for mellanox platform you have 2700, 2100, and etc. on. are all the platforms to share a same one image or they have different images? It will be better for sonic to produce less images and each image can be shared by multiple hwskus. Then, in this case, the SKU is probably not useful at the build time now.